app/services/widgets/widgets.ts

```diff
@@ -220,6 +218,26 @@ export class WidgetsService extends StatefulService<IWidgetSourcesState> impleme
     });
   }
 
+  private register(sourceId: string) {
+    const source = this.sourcesService.getSource(sourceId);
+    const widgetType = source.getPropertiesManagerSettings().widgetType;
+    const isWidget = typeof widgetType === 'number';
```

Comment:
Realize this is copied from above, but it's probably safer / more correct to just use `source.getPropertiesManagerType()` instead.